config_files/config_de_novo_uppmax.yaml

```diff
@@ -1,7 +1,7 @@
 Pipelines:
  QCcontrol: ["fastqc", "abyss", "trimmomatic", "kmergenie", "align"]
  assemble: ["abyss", "allpaths", "cabog",  "masurca", "soapdenovo", "spades", "trinity"]
- evaluete: ["align", "qaTools", "FRC"]
+ evaluete: ["align", "qaTools", "FRC", "BUSCObacteria", "BUSCOeukarya", "BUSCOvertebrata", "BUSCOfungi"]
  align: []
```

Comment:
@remiolsen why this choice? It is not more natural to have the dataset to test (bacteria, eukariota, etc) as a parameter to be set in the sample configuration file rather than having it as a built in pipeline?